tutorials/cloud/aws/SageMaker_ASR_Training.ipynb

```diff
@@ -0,0 +1,418 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e17535a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "line = \"from nemo.utils.cloud import initialize_sagemaker; initialize_sagemaker()\"\n",
+    "with open(code_dir / \"speech_to_text_ctc.py\", 'r+') as f:\n",
+    "    content = f.read()\n",
+    "    f.seek(0, 0)\n",
+    "    f.write(line.rstrip('\\r\\n') + '\\n' + content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bfa2199e",
+   "metadata": {},
+   "source": [
+    "### 3. Setup the AN4 Dataset, upload data to S3\n",
+    "\n",
+    "We now download our training and validation data, uploading to S3 so that SageMaker can mount our data to the instance at runtime."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1c5a60a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# within the SageMaker container, mount_dir will be where our data is stored.\n",
+    "download_an4(\n",
+    "    data_dir=str(data_dir),\n",
+    "    train_mount_dir=\"/opt/ml/input/data/training/\",\n",
+    "    test_mount_dir=\"/opt/ml/input/data/testing/\",\n",
+    ")\n",
+    "\n",
+    "# Upload to the default bucket\n",
+    "prefix = \"an4\"\n",
+    "bucket = sess.default_bucket()\n",
+    "loc = sess.upload_data(path=str(data_dir), bucket=bucket, key_prefix=prefix)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b1de1089",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_path = \"s3://\" + sess.default_bucket() + \"/nemo-output/\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6321e3a9",
+   "metadata": {},
+   "source": [
+    "### 4. Configure the training job\n",
+    "\n",
+    "Now we configure the training job, by modifying the `config.yaml` file that is stored in our source code directory.\n",
+    "We pass relative directory paths for the data based on the SageMaker mount directory on the remote instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4bb61640",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conf = OmegaConf.load(config_path)\n",
+    "\n",
+    "# Set Data Locations based on the mounted directory in the SageMaker instance\n",
+    "conf.model.train_ds.manifest_filepath = \"/opt/ml/input/data/training/an4/train_manifest.json\"\n",
+    "conf.model.validation_ds.manifest_filepath = \"/opt/ml/input/data/testing/an4/test_manifest.json\"\n",
+    "conf.trainer.accelerator = \"gpu\"\n",
+    "conf.trainer.max_epochs = 150\n",
+    "\n",
+    "# Output directory for our experiment within the SageMaker instance\n",
+    "conf.exp_manager.exp_dir=\"/opt/ml/model/\"\n",
+    "\n",
+    "# Create a Small Variant of the Conformer Model\n",
+    "conf.model.encoder.n_layers = 8\n",
+    "conf.model.n_heads = 4\n",
+    "conf.model.spec_augment.time_masks = 5\n",
+    "\n",
+    "# Set Optimizer parameters\n",
```

Comment:
Lets note that LR here is a scaler multiplier for Noam, not actually LR of 2.0 (we've had comments before)

Reply:
I didn't even know this, thanks for clarification! added.